channeldb/channel.go

```diff
@@ -138,6 +139,11 @@ type OpenChannel struct {
 	// channel as on-chain conditions change.
 	MinFeePerKb btcutil.Amount
 
+	// DustLimit is the threshold below which no HTLC output should be
+	// generated for this node’s commitment transaction; ie. HTLCs below
+	// this amount are not enforceable onchain.
+	DustLimit btcutil.Amount
```

Comment:
As I described below I don't separate the dust limit on _their_ and _our_ , the dust limit which I store it is negotiated value and it should be identical for both sides.